site/en/docs/webstore/about_webstore/index.md

```diff
@@ -6,33 +6,24 @@ description: >
   An explanation of the Chrome Web Store and why you might want to use it.
 ---
 
-The [Chrome Web Store][1] lets you publish apps and games—either free or paid—where Google Chrome
-users can easily find them. Whenever Chrome users create a new window or tab, they see the apps
-they've installed, along with a store icon that reminds them to find more apps.
+The [Chrome Web Store][1] lets you publish apps and extensions&mdash;either free or paid&mdash;where Google Chrome
+users can easily find them. 
 
-If you already have a web app, you can create a metadata file and then publish your app in the store
-within minutes. Or you can create and publish an app that's specialized for Chrome, such as an
-extension for Chrome or a web app that's been modified to be downloaded and run on the user's
-computer.
+Your published item can be free, or you can charge for it using any payment system you like.
 
-Your app can be free, or you can charge for it using either Chrome Web Store Payments or your own
-custom payment system. A [Licensing API][2] lets you check whether the current user has paid for
-your app with Chrome Web Store Payments.
-
-To upload and publish apps, use the [Chrome Developer Dashboard][3].
+To upload and publish items, use the [Chrome Developer Dashboard][3].
 
 ## Why should I use the Chrome Web Store?
 
-Publishing your app in the Chrome Web Store lets you reach more people. You can also earn revenue or
-build Chrome-specific features into your app.
+Publishing your app or extension in the Chrome Web Store lets you reach more people. You can also earn revenue or
+build Chrome-specific features into it.
 
 ## How do I start?
 
 1.  Read the [technical overview][4]
-2.  Publish your first app by following the [Getting Started tutorial][5]
+2.  Publish your first extension by following the [Getting Started tutorial][5]
 
 [1]: http://chrome.google.com/webstore
-[2]: /docs/webstore/check_for_payment
 [3]: https://chrome.google.com/webstore/developer/dashboard
 [4]: /docs/webstore/overview
 [5]: /docs/webstore/get_started_simple
```

Comment:
/docs/webstore/get_started_simple is being removed in this PR.

Reply:
Good catch, fixed.